bibliopixel/animation/remote/control.py

```diff
@@ -172,7 +179,10 @@ def run_animation(self, name=None):
             name = self.default
 
         if name not in self.name_map:
-            return False, 'Invalid animation name: {}'.format(name)
+            error = 'Invalid animation name: {}'.format(name)
+            log.info(error)
+            print('!!!!!', self.name_map)
```

Comment:
Left over debug print? We shouldn't have *any* print() statements in the code. Just log.<something>

Reply:
Urg!  :-D 

Pushed a fix.